TTS_xwing/src/Game/Mechanic/Movement/Maneuver.ttslua

```diff
@@ -0,0 +1,242 @@
+require("TTS_lib.Util.Table")
+local Dim = require("TTS_xwing.src.Dim")
+
+local igu = Dim.Convert_mm_igu
+-- TODO: add reverse move, and reverse bank
+
+----------------------------------------
+-- Standard X-Wing Maneuver Templates --
+----------------------------------------
+
+local XWingTemplates = {
+    straight = { 40, 80, 120, 160, 200 }, -- speed to millimeters
+    turn = { 35, 63, 90 },                -- Turn Middle Radius
+    bank = { 80, 130, 180 },              -- Bank Middle Radius
```

Comment:
Are these work in progress?
Did you take these from somewhere?
Do they align?

Reply:
Taken from here:
https://www.reddit.com/r/XWingTMG/comments/41slns/component_dimensions/

I've found no better source.